naming/src/main/java/com/alibaba/nacos/naming/controllers/InstanceController.java

```diff
@@ -126,10 +127,7 @@ public String update(HttpServletRequest request) throws Exception {
         String serviceName = WebUtils.required(request, CommonParams.SERVICE_NAME);
         String namespaceId = WebUtils.optional(request, CommonParams.NAMESPACE_ID, Constants.DEFAULT_NAMESPACE_ID);
 
-        String agent = request.getHeader("Client-Version");
-        if (StringUtils.isBlank(agent)) {
-            agent = request.getHeader("User-Agent");
-        }
+        String agent = request.getHeader(HttpHeaderConsts.USER_AGENT_HEADER);
```

Comment:
Ditto below Suggestions, here can be changed to a similar Map. getDefault (request, key, defaultKey) strategy